libsql-server/src/namespace/mod.rs

```diff
@@ -803,6 +804,17 @@ impl<T: Database> Namespace<T> {
         self.db.shutdown().await?;
         Ok(())
     }
+
+    pub fn jwt_key(&self) -> crate::Result<Option<jsonwebtoken::DecodingKey>> {
+        let config = self.db_config_store.get();
+        if let Some(jwt_key) = config.jwt_key.as_deref() {
+            Ok(Some(
+                parse_jwt_key(jwt_key).context("Could not parse JWT decoding key")?,
```

Comment:
It would be best if you decoded the key only once, when the namespace is loaded. As per jsonwebtoken Documentation:
> All the different kind of keys we can use to decode a JWT. This key can be re-used so make sure you only initialize it once if you can for better performance.

Also, consider wrapping it in an `Arc`: cloning the key is not cheap.

Reply:
I was thinking about it but didn't have a good place to store the key. Let's do this in a follow up as this PR is hanging for some time now.